packages/server/tests/acceptance/rateLimiter.spec.ts

```diff
@@ -54,22 +53,19 @@ describe('@ratelimiter Rate Limiters Acceptance Tests', function () {
 
     describe('RPC Rate Limiter Acceptance Tests', () => {
         it('should throw rate limit exceeded error', async function () {
-            let rateLimited = false;
-            try {
-                //Currently chaindId is TIER 2 request per LIMIT_DURATION from env. We are trying to get an error for rate limit by exceeding this threshold
+            const sendMultipleRequests = async () => {
                 for (let index = 0; index < TIER_2_RATE_LIMIT * 2; index++) {
                     await relay.call(testConstants.ETH_ENDPOINTS.ETH_CHAIN_ID, [null], requestId);
                     // If we don't wait between calls, the relay can't register so many request at one time. So instead of 200 requests for example, it registers only 5.
                     await new Promise(r => setTimeout(r, 1));
                 }
-            } catch (error) {
-                rateLimited = true;
-                Assertions.jsonRpcError(error, predefined.IP_RATE_LIMIT_EXCEEDED(testConstants.ETH_ENDPOINTS.ETH_CHAIN_ID));
             }
+            const error = predefined.IP_RATE_LIMIT_EXCEEDED(testConstants.ETH_ENDPOINTS.ETH_CHAIN_ID)
```

Comment:
Nit: semi-colon can be added at the end of this line as well as the line above it.